modules/odc-adding-components-to-an-existing-project.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-viewing-application-composition-using-topology-view.adoc
+
+[id="odc-adding-components-to-an-existing-project_{context}"]
+= Adding components to an existing project
+
+.Procedure
+
+. Click *Add to Project* (image:odc_add_to_project.png[title="Add to Project"]) or press kbd:[Ctrl+Space]
+. Search the component and select *Create* or press kbd:[Enter] to add the component in the topology.
```

Comment:
Search for the component...to add the component to the project and see it in the topology **Graph view**.